pkg/machine/ignition.go

```diff
@@ -506,14 +534,14 @@ func prepareCertFile(path string, name string) (File, error) {
 	return file, nil
 }
 
-func getProxyVariables() string {
-	proxyOpts := ""
+func GetProxyVariables() *map[string]string {
```

Comment:
Why pointer to a map? Maps are already reference types

Reply:
Good notice, thanks. Fixed